src/components/PreviewDimension.tsx

```diff
@@ -0,0 +1,264 @@
+import React, { useState, ChangeEvent, useEffect } from "react";
+
+import { Card, Typography, Tooltip } from "antd";
+import Dimension from "../components/Dimension";
+import { Dimension as DimensionType } from "../types/dimension";
+
+import edit from "../assets/images/edit.svg";
+import save from "../assets/images/save.png";
+import cancel from "../assets/images/cancel.png";
+import "../styles/DisplayCards.css";
+
+interface PreviewDimensionProps {
+  fullDimensionView: boolean;
+  dimension: DimensionType;
+  previewSliderPosChange: (value: number, dimensionKey: string) => void;
+  saveDimensionFunction: (updatedDimension: DimensionType) => void;
+  openSingleDimension: (key: string) => void;
+  saveDimensionClicked: boolean;
+}
+
+export enum CardSide {
+  Left,
+  Right,
+}
```

Comment:
I think this should be imported from DisplayCards instead defined again

Reply:
okay, I'll just refactor out all the common things tomorrow then!